loadtest/store/store.go

```diff
@@ -49,15 +49,16 @@ type UserStore interface {
 	Reactions(postId string) ([]model.Reaction, error)
 
 	// Random things
-	// RandomChannel returns a random channel for a user.
-	RandomChannel(teamId string) (model.Channel, error)
-	// RandomChannelJoined returns a random channel for the given teamId that the
-	// current user is a member of.
-	RandomChannelJoined(teamId string) (model.Channel, error)
-	// RandomTeam returns a random team for a user.
-	RandomTeam() (model.Team, error)
-	// RandomTeamJoined returns a random team the current user is a member of.
-	RandomTeamJoined() (model.Team, error)
+	// RandomAnyChannel returns a random channel for a user.
+	RandomAnyChannel(teamId string) (model.Channel, error)
+	// RandomChannel returns a random channel for the given teamId that the
+	// current user is a member of or not a member of.
+	RandomChannel(teamId string, memberOf bool) (model.Channel, error)
```

Comment:
That's very clever! I think we can completely remove `RandomAnyChannel`. I am not sure if we need to define constants because our primary use case is `RandomChannel(id, true)` and it is adequately self descriptive IMO :)

Reply:
Not a huge fan of *clever* code like that actually but I guess it will do for now at least to keep the number of methods down a bit. Also as you pointed out it won't be very frequent.

Reply:
Actually, it's not very apparent to me what `true` means just by looking at `RandomChannel(id, true)`. However, `RandomChannel(id, store.SelectMemberOf)` is immediately clear. At the expense of a little more verbosity I would suggest to use a const.

Let's also use bit shifts for this like `= 1 << iota`. That allows us to do `SelectAny = SelectMemberOf | SelectNotMemberOf` :)

Reply:
When this is merged I am also going to add a `SelectNotCurrent` which is needed when switching to a channel/team and we obviously want to make sure it is not the same we are currently viewing. So the const approach gives us way more flexibility which is what we need right now.